src/main/java/io/kaitai/struct/ByteBufferKaitaiStream.java

```diff
@@ -114,22 +113,19 @@ public ByteBuffer asRoBuffer() {
      * references to it, which breaks all subsequent <code>read..</code> methods with
      * {@link NullPointerException}. Afterwards, a call to {@link System#gc()} will
      * typically release the mmap, if garbage collection will be triggered.
-     * </p>
      * <p>
      * There is a <a href="https://bugs.java.com/bugdatabase/view_bug.do?bug_id=4724038">
      * JDK-4724038 request for adding unmap method</a> filed at Java bugtracker since 2002,
      * but as of 2018, it is still unresolved.
-     * </p>
      * <p>
      * A couple of unsafe approaches (such as using JNI, or using reflection to invoke JVM
      * internal APIs) have been suggested and used with some success, but these are either
      * unportable or dangerous (may crash JVM), so we're not using them in this general
      * purpose code.
-     * </p>
      * <p>
      * For more examples and suggestions, see:
-     * https://stackoverflow.com/questions/2972986/how-to-unmap-a-file-from-memory-mapped-using-filechannel-in-java
-     * </p>
+     * <a href="https://stackoverflow.com/questions/2972986/how-to-unmap-a-file-from-memory-mapped-using-filechannel-in-java">How to unmap a file from memory mapped using FileChannel in java?</a>
```

Comment:
If we're already touching this, the SO link can be shortened:

```suggestion
     * <a href="https://stackoverflow.com/q/2972986">How to unmap a file from memory mapped using FileChannel in java?</a>
```

Reply:
> If we're already touching this, the SO link can be shortened

The IntelliJ IDEA wasn't happy to see a link in plain text, so I thought I would fix it when I'm already doing changes around, even if it's not strictly relevant to the PR topic.